apps/frontend/src/app/5_pages/MarketMakingPage/constants/mainnet.ts

```diff
@@ -0,0 +1,16 @@
+import { SupportedTokens } from '@sovryn/contracts';
+import { ChainIds } from '@sovryn/ethers-provider';
+
+import { PromotionColor } from '../MarketMakingPage.types';
+import { AmmLiquidityPool } from '../utils/AmmLiquidityPool';
+
+export const mainnetAmm = [
```

Comment:
This doesn't follow our naming conventions (should be `MAINNET_AMM`) and we also don't use this folder style (you should just create `MarketMakingPage.constants.ts` and place it there). Do this for `constants/testnet` as well.